.rubocop.yml

```diff
@@ -14,10 +14,6 @@ inherit_from:
 
 AllCops:
   NewCops: enable
-  Exclude:
-    - "bin/*"
-    - "db/schema.rb"
-    - "node_modules/**/*"
```

Comment:
I think we **want** to exclude these?

Reply:
Why? 🤔

Reply:
We don't care about Rubocop checking node modules as it's just JS stuff, same for bin, there's some usually some shell scripts in there. Db schema maybe?

Reply:
cool 👍🏼 - dropped the commit.